trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -145,16 +145,43 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+This section uses the Augmented Backus-Naur Form (ABNF) notation of [RFC5234](https://tools.ietf.org/html/rfc5234), including the DIGIT rule from that document. It also includes the OWS rule from [RFC7230](https://www.rfc-editor.org/info/rfc7230).
+
+```
+dictionary  = dict-member 0*128( OWS "," OWS dict-member )
+dict-member = key "=" value
+```
+
+Identifiers are short (up to 256 characters) textual identifiers.
 
-The value a concatenation of trace graph name-state pairs. Only one entry per
-name is allowed because the entry represents that last position in the trace.
+```
+key = lcalpha 0*255( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
+lcalpha    = %x61-7A ; a-z
+```
+
+Note that identifiers can only contain lowercase letters.
```

Comment:
Recommend changing this line to: 

Note that keys must begin with a lowercase letter, and can only contain lowercase letters `a-z`, digits `0-9`, underscores `_`, dashes `-`, asterisks `*`, and forward slashes `/`.